sdk/tables/azure-data-tables/pom.xml

```diff
@@ -31,7 +32,7 @@ Licensed under the MIT License.
     <tag>HEAD</tag>
   </scm>
 
-  <!-- CosmosTableSkip - Needed temporary values to 10% not fail. -->
+  <!-- CosmosTableSkip - Needed temporary values to 20% not fail. -->
```

Comment:
I'm not trying to be a nag but this isn't setting a temporary value for coverage percentage, it's skipping the check altogether. Please either update the comment or the property to put them in sync.